providers/amazon/src/airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -270,7 +274,7 @@ def display_logs_from(log_group: str, continuation_token: str | None) -> str | N
                 self.log.info("No new log from the Glue Job in %s", log_group)
             return next_token
 
-        log_group_prefix = self.conn.get_job_run(JobName=job_name, RunId=run_id)["JobRun"]["LogGroupName"]
+        log_group_prefix = job_run["LogGroupName"]
```

Comment:
Ah, nice, I'm glad you can make that one call multi-purpose down here too.  That works out nicely.

Reply:
I was thinking of making it a property. But, I'll see how often its used across and do refactoring as part of another PR